### PR TITLE
Add JChem

### PR DESCRIPTION
**Pull request by mcs07**
JChem using a jython script.

### CODE REVIEW

**Comment by baoilleach** [comment]
Could you make a change to the HydrogenCounter?  While it's possible to perfectly count the hydrogens without doing kekulization, one of the outputs of the benchmark is to find examples of "could do better" wrt kekulization.

I think the way to do this is to add "success = molecule.dearomatize()" (found at https://docs.chemaxon.com/display/docs/Converting+structure+from+aromatic+form+to+Kekule+form), and return (None, "Kekulization_failure") if success is False.

**Comment by baoilleach** [comment]
(Bump.) Would be great...?

File: scripts/JChem.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            return ''
+        return msmi
+
+
+class MyHydrogenCounter(common.HydrogenCounter):
+    def getoutput(self, smi):
+        mol = MolImporter.importMol(smi, 'smiles')
+        success = mol.dearomatize()
+        if not success:
+            return (None, 'Kekulization_failure')
```
Comment (mcs07) [inline scripts/JChem.py:39]: Does this do what you want? Was added in https://github.com/nextmovesoftware/smilesreading/pull/1/commits/fd8857ff2e0c2e1a4b3da62e6b3c2fd75a47285d

**Comment by baoilleach** [comment]
Ah right. Apolgies. I didn't realise you had done this. The github page I'm writing this on seems to think that these commits were done on 15 Sep, i.e. before my comment.

**Comment by mcs07** [comment]
I think I might have confused GitHub a bit by force-pushing a revised history to avoid bloating the repository with unneeded .gz result files. The dates on the individual commits all look correct though.

**Pull request merged**